### PR TITLE
feat: CD enablement

### PR DESCRIPTION
**Pull request by candiduslynx**
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -34,8 +34,9 @@
         <url>https://github.com/${gitHubRepo}</url>
     </scm>
     <properties>
-        <revision>2.4.1</revision>
-        <changelist>-SNAPSHOT</changelist>
+        <revision>3</revision>
+        <changelist>999999-SNAPSHOT</changelist>
+        <hpi.compatibleSinceVersion>3</hpi.compatibleSinceVersion>
```
Comment (NotMyFault) [inline pom.xml]:
```suggestion
```
This is not needed for CD, nor needed in this case. There are no incompatible config changes users can not upgrade from.

Comment (candiduslynx) [reply pom.xml]: there was an incompatibility for ver 2.0.0, and as now the versioning changes i wanted to restore this (I missed this here: https://github.com/jenkinsci/zos-connector-plugin/pull/19/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8

Comment (candiduslynx) [reply pom.xml]: Namely, this line: https://github.com/jenkinsci/zos-connector-plugin/blob/02b64c4947aa240bcf42d9e025b6eb12188e3531/pom.xml#L62

**Review by NotMyFault — APPROVED** [review]

**Comment by NotMyFault** [comment]
Thanks for the clarification, the PR looks good so far. Bear in mind you need to add a label like 'enhancement' in order to pick this PR up.

**Pull request merged**